cmd/lit-af/shell.go

```diff
@@ -331,7 +361,7 @@ func printHelp(commands []*Command) {
 func printCointypes() {
 	for k, v := range coinparam.RegisteredNets {
 		fmt.Fprintf(color.Output, "CoinType: %s\n", strconv.Itoa(int(k)))
-		fmt.Fprintf(color.Output, "└────── Name: %s,\tBech32Prefix: %s\n", v.Name, v.Bech32Prefix)
+		fmt.Fprintf(color.Output, "└────── Name: %s,%sBech32Prefix: %s\n\n", v.Name, strings.Repeat(" ", 13-len(v.Name)), v.Bech32Prefix)
```

Comment:
Alternatively, i think you can use `%-13s` in the format string and append the comma to v.Name in the parameters:

```
fmt.Fprintf(color.Output, "└────── Name: %-13sBech32Prefix: %s\n\n", v.Name + ",", v.Bech32Prefix)
```

Reply:
Thx, this looks a bit ugly, yeah